repositories/ResumeRepository.ts

```diff
@@ -1,8 +1,22 @@
 import { ResumeModel } from 'models/ResumeModel';
```

Comment:
I've found that VSCode's auto-importer fails when using absolute routes. Make sure you use the relative route like `../models/ResumeModel` whenever you're importing.